src/test/scala/mesosphere/mesos/ResourceMatcherTest.scala

```diff
@@ -134,7 +134,7 @@ class ResourceMatcherTest extends UnitTest with Inside with TableDrivenPropertyC
 
       val resourceMatchResponse = ResourceMatcher.matchResources(offer, app, knownInstances = Seq.empty, unreservedResourceSelector, config, Seq.empty)
 
-      resourceMatchResponse shouldBe a[ResourceMatchResponse.Match]
+      resourceMatchResponse.should(be(a[ResourceMatchResponse.Match]))
```

Comment:
I had to rewrite these expressions to not use infix notation. We hit a weird Scala compiler bug with it that will likely be fixed in the soon-to-be-released Scala 2.13.2.